def/babel6.js

```diff
@@ -10,47 +10,49 @@ module.exports = function (fork) {
     var or = types.Type.or;
 
     def("Directive")
+        .bases("Node")
         .build("value")
         .field("value", def("DirectiveLiteral"));
 
     def("DirectiveLiteral")
+        .bases("Node", "Expression")
         .build("value")
         .field("value", String, defaults["use strict"]);
 
     def("BlockStatement")
         .bases("Statement")
         .build("body")
         .field("body", [def("Statement")])
-        .field("directives", [def("Directive")]); // add directive
+        .field("directives", [def("Directive")], defaults.emptyArray);
 
     def("Program")
         .bases("Node")
         .build("body")
         .field("body", [def("Statement")])
-        .field("directives", [def("Directive")]); // add directive
+        .field("directives", [def("Directive")], defaults.emptyArray);
 
     // Split Literal
     def("StringLiteral")
-        .bases("Literal")
+        .bases("Node", "Expression")
```

Comment:
`Node` is not important here, nor for the other Babel `*Literal` types, because `Expression <: Node`.